### PR TITLE
[1.10 Only] Use new Hub workload pool

### PR DESCRIPTION
**Pull request by JimmyCYJ**
This PR includes https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/694.

Roll forward https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/678 with some fixes to GKE_CLUSTER_URL

### CODE REVIEW

File: asm/Kptfile [hunk 1]
```diff
@@ -212,7 +212,7 @@ openAPI:
       x-k8s-cli:
         substitution:
           name: token-audiences-hub
-          pattern: "istio-ca,ENVIRON_PROJECT_ID.hub.id.goog"
+          pattern: "istio-ca,ENVIRON_PROJECT_ID.svc.id.goog"
```
Comment (jasonwzm) [inline asm/Kptfile:215]: Can we make these changes in https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/694?

**Review by jasonwzm — CHANGES_REQUESTED** [review]

File: asm/Kptfile (hunk 1 above)
Comment (JimmyCYJ) [reply asm/Kptfile:215]: Done.

**Comment by jasonwzm** [comment]
@JimmyCYJ Can you rebase and retest?

**Review by jasonwzm — APPROVED** [review]

**Comment by JimmyCYJ** [comment]
This PR will merge on Monday

**Comment by JimmyCYJ** [comment]
Test keeps failing with at downloading step
```
install_asm: Checking installation tool dependencies...
install_asm: Authorizing asm-installer@asm-scriptaro-oss.iam.gserviceaccount.com with /workspace/scripts/asm-installer/key.json...
install_asm: Fetching/writing GCP credentials to kubeconfig file...
install_asm: [WARNING]: nc not found, skipping k8s connection verification
install_asm: [WARNING]: (Installation will continue normally.)
install_asm: Downloading ASM..
install_asm: [WARNING]: Failed, retrying...(1 of 2)
install_asm: [WARNING]: Failed, retrying...(2 of 2)
```

Opened https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/700 with verbose output.

**Pull request merged**